modules/020-deckhouse/docs/README.md

```diff
@@ -24,3 +24,30 @@ In Deckhouse, this module sets up:
 - **Service for validating Custom Resources**.
 
   The validation service prevents creating Custom Resources with invalid values or adding such values to the existing Custom Resources. Note that it only tracks Custom Resources managed by Deckhouse modules.
+
+
+### DeckhouseReleases
+
+You can get Deckhouse releases list via the command `kubectl get DeckhouseRelease`. By default, cluster keeps last 10 outdated release and all deployed/pending releases
+
+Every release can have one of the next statuses:
+`Pending` - release is waiting to be deployed: waiting for update window, canary deployment, etc. You can see the detailed status via `kubectl describe DeckhouseRelease $name` command
+`Deployed` - release is applied. It means that Deckhouse image tag was changed, but the update process of the all components
+is going asynchronously and could have not been finished yet
+`Outdated` - release is outdated and not used anymore
+`Suspended` - release was suspended (for ex. it has an error). Can be set only if `suspended` release was not deployed yet
+
+---
+
+##### Update process
+When release status is changed to `Deployed` state, release is updating only a tag of the Deckhouse image.
+Deckhouse will start checking and updating process of the all modules, which were changed from the last release.
+Duration of an update could be different and connected to cluster size, enabled modules count and settings.
+Foe example: if you cluster have a lot of `NodeGroup` resources, it will take some time to update them because this resources are updated one by one
```

Comment:
```suggestion
For example: if your cluster has a lot of `NodeGroup` resources, it will take some time to update them because these resources are updated one by one
```